rosdep/python.yaml

```diff
@@ -5931,6 +5931,10 @@ python3-pyqt5.qtwebengine:
   debian: [python3-pyqt5.qtwebengine]
   fedora: [python3-qt5-webengine]
   ubuntu: [python3-pyqt5.qtwebengine]
+python3-pyswarms-pip:
+  ubuntu:
```

Comment:
Can you please add equivalent pip rules for debian and fedora?  Thanks.

Reply:
I'm quite new to this, so hopefully I'm right in assuming the rules are exactly the same debian and fedora... if so, done!